File: src/main/java/com/rwbymod/z_proxy/CommonProxy.java
package com.rwbymod.z_proxy;

import com.rwbymod.block.BlockRegistry;
import com.rwbymod.item.ItemRegistry;
import com.rwbymod.misc.CraftingRegistry;
import com.rwbymod.misc.MiscRegistry;

public class CommonProxy {

	public void preInit() {
		MiscRegistry.init();
		ItemRegistry.init();
		BlockRegistry.init();
		CraftingRegistry.init();
	}
	
	public void init() {}
	public void postInit() {}

}

File: src/main/scala/com/rwbymod/z_proxy/CommonProxy.scala
package com.rwbymod.z_proxy

import net.minecraftforge.fml.common.network.NetworkRegistry
import com.rwbymod.RWBYMod
import com.rwbymod.registry.MiscRegistry
import com.rwbymod.registry.ItemRegistry
import com.rwbymod.registry.BlockRegistry
import com.rwbymod.registry.CraftingRegistry
import com.rwbymod.gui.GuiHandler

class CommonProxy {

  def preInit {
    MiscRegistry.init
    ItemRegistry.init
    BlockRegistry.init
    CraftingRegistry.init
  }

  def init {
    NetworkRegistry.INSTANCE.registerGuiHandler(RWBYMod.instance, new GuiHandler())
  }

  def postInit {}

}